src/platform/site-wide/user-nav/selectors.js

```diff
@@ -13,6 +14,16 @@ export const selectUserGreeting = createSelector(
       return startCase(toLower(name.first || sessionFirstName));
     }
 
-    return 'My Account';
+    return [
+      <span key="show-for-small-only" className="show-for-small-only">
+        My Account
+      </span>,
+      <span
+        key="show-for-medium-up"
+        className="user-dropdown-email show-for-medium-up"
+      >
+        {email}
+      </span>,
+    ];
```

Comment:
@ncksllvn Made the changes you suggested. Heer is my solution for it. I had to use a span because it worked correctly with the foundation responsive utility classes, without having to fiddle with the css too much.

Reply:
@ncksllvn @jbalboni can I gets some eye's on this so I can close this one out.